JDI.Light/JDI.Light/Elements/Complex/Table/Line.cs

```diff
@@ -18,23 +18,25 @@ public Line(List<string> list, List<string> headers)
             _headers = headers;
         }
 
-        public Line(List<UIElement> elements, List<string> headers)
+        public Line(IEnumerable<IWebElement> elements, List<string> headers)
```

Comment:
Why do we use IWebElement everywhere? UIElement gives us more functionality here.